client/packages/common/src/ui/components/buttons/standard/DialogButton.tsx

```diff
@@ -45,6 +48,12 @@ const getButtonProps = (
         labelKey: 'button.cancel',
         variant: 'outlined',
       };
+    case 'back':
+      return {
+        icon: <ArrowLeftIcon />,
+        labelKey: 'button.back',
+        variant: 'outlined',
+      };
```

Comment:
Figured that "Back" was a generic enough button to be included here in the common set. Kept "Next Step" as a "custom" button, but could add here if we thought it warranted it?

Reply:
I think `Next Step` isn't really use case specific - it's pretty generic to any kind of stepper modal. However it seems like the talk is leaning away from the modal stepper situation, so happy to leave as custom for now, and can refactor later once the design decisions are clearer.